src/AF_Packet.cc

```diff
@@ -1,6 +1,13 @@
 
 #include "zeek/zeek-config.h"
 
+// Very recent Zeek versions do not provide ZEEK_VERSION_NUMBER through
+// zeek-config.h when compiled as included/builtin/static plugin. These
+// are required to explicitly include zeek/zeek-version.h.
+#ifndef ZEEK_VERSION_NUMBER
+#include "zeek/zeek-version.h"
+#endif
```

Comment:
This seems fragile and I think just checking whether the header exists and then including it should be more direct, e.g.,

```suggestion
#if __has_include("zeek/zeek-version.h")
#include "zeek/zeek-version.h"
#endif
```